docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -947,6 +948,14 @@ When using `host` networking via `Network=host`, the `PublishPort=` option canno
 
 This key can be listed multiple times.
 
+
+### `ServiceName=`
```

Comment:
I thought about it as well. But then I thought:
- The service for `.container` and `.kube` files keeps the name
- Unit types `.network`, `.volume`, `.image` and `.build` are intended mostly for dependencies
- `.pod` are the only ones that both the users interact with and the name is not kept.

Plus, it making these change for other types is more complicated because it requires handling the order differently. Something along the line of what I did for `.pod` files.

Reply:
I guess I am open to wait for user feedback on this, but generally I find doing this for one option inconsistent and I am pretty sure that someone will request this for other types as well.
(I do agree that kube and container units are fine without it as they do not append anything)